shared/dates.ts

```diff
@@ -5,10 +5,13 @@ export const DATE_FNS_INPUT_FORMAT = 'yyyy-MM-dd'; // example: 1969-07-16
 export const DATE_FNS_LONG_OUTPUT_FORMAT = 'd MMMM yyyy'; // example: 1 July 2021
 export const DATE_FNS_SHORT_OUTPUT_FORMAT = 'd MMM yyyy'; // example: 5 Jan 2019
 
-export const cancellationFormatDate = (cancellationEffectiveDate?: string) => {
+export const cancellationFormatDate = (
+	cancellationEffectiveDate?: string,
+	outputFormat: string = DATE_FNS_SHORT_OUTPUT_FORMAT,
```

Comment:
Just curious chat does FNS stand for?

Reply:
`date-fns` is the name of the date formatting library being used. I don't know why that needs to be in the name of these constants, but I'm simply reusing what was already there so haven't renamed then.